hyke/api/jobs_system.py

```diff
@@ -23,82 +23,40 @@
 logger = get_logger(__name__)
 
 
-def scheduled_system():
+def get_hike_system_scheduled():
     print("Scheduled task is started for Hyke System...")
 
-    items = StatusEngine.objects.filter(Q(outcome=-1) & Q(formationtype__startswith="Hyke System"))
+    items = StatusEngine.objects.filter(
+        Q(outcome=StatusEngine.SCHEDULED) & Q(formationtype__startswith="Hyke System")
```

Comment:
There was a check on most statements below if the outcome was scheduled.  I removed them because this query already ensured that